zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/metrics/DbUsageMetricStateTest.java

```diff
@@ -101,4 +103,34 @@ public void shouldRecordEDIMetrics() {
         .containsExactlyInAnyOrderEntriesOf(
             Map.of(TenantOwned.DEFAULT_TENANT_IDENTIFIER, 2L, "tenant1", 3L, "tenant2", 1L));
   }
+
+  @Test
+  public void shouldRecordTUMetrics() {
+    // given
+    final var eventTime = InstantSource.system().millis();
+    when(mockClock.millis()).thenReturn(eventTime);
+    state.resetActiveBucket(1L);
+
+    // when
+    state.recordTUMetric(TenantOwned.DEFAULT_TENANT_IDENTIFIER, "assignee1");
```

Comment:
🔧 You could add the same user twice just to confirm it only tracks it once. It is stored in a `Set` so it obviously will but still, maybe in the future someone decides to change it to `List` and than this would fail.